### PR TITLE
(feat) publish docker data to create drone card

### PR DESCRIPTION
**Pull request by eoinmcafee00**
On merge - update page setting to point to master & not branch

### CODE REVIEW

File: docker.go [hunk 1]
```diff
@@ -204,6 +297,17 @@ func commandPull(repo string) *exec.Cmd {
...
 
+func commandInspect(build Build, tag string) *exec.Cmd {
+	target := fmt.Sprintf("%s:%s", build.Name, tag)
+	args := []string{
+		"docker inspect",
+	}
+	args = append(args, target)
+	args = append(args, "> /tmp/output.json")
+
+	return exec.Command(bashExe, "-c", strings.Join(args, " "))
```
Comment (shubham149) [inline docker.go:308]: bash won't be present in windows image. Also, /tmp directory would be present in windows.

Comment (shubham149) [reply docker.go:308]: Use this command instead:
`
exec.Command("docker", "inspect")
`
and redirect stdout and stderr to /tmp/output.json file

Comment (eoinmcafee00) [reply docker.go:308]: @shubham149 - I missed your updates where you've added in windows support, which is the reason this was missed. Will look into it

Comment (bradrydzewski) [reply docker.go:308]: instead of using bash and writing to a file, you can read directly from `exec.Cmd.Stdout`

Comment (eoinmcafee00) [reply docker.go]: yeah this has been cleaned up.

**Review by bradrydzewski — APPROVED** [review]

**Pull request merged**